xbmc/GUIInfoManager.cpp

```diff
@@ -6871,217 +6871,217 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
-    {"hasarchive", LISTITEM_HASARCHIVE},
-    {"inprogress", LISTITEM_INPROGRESS},
-    {"isencrypted", LISTITEM_ISENCRYPTED},
-    {"progress", LISTITEM_PROGRESS},
-    {"dateadded", LISTITEM_DATE_ADDED},
-    {"dbtype", LISTITEM_DBTYPE},
-    {"dbid", LISTITEM_DBID},
-    {"appearances", LISTITEM_APPEARANCES},
-    {"stereoscopicmode", LISTITEM_STEREOSCOPIC_MODE},
-    {"isstereoscopic", LISTITEM_IS_STEREOSCOPIC},
-    {"imdbnumber", LISTITEM_IMDBNUMBER},
-    {"episodename", LISTITEM_EPISODENAME},
-    {"timertype", LISTITEM_TIMERTYPE},
-    {"epgeventtitle", LISTITEM_EPG_EVENT_TITLE},
-    {"epgeventicon", LISTITEM_EPG_EVENT_ICON},
-    {"timerisactive", LISTITEM_TIMERISACTIVE},
-    {"timerhaserror", LISTITEM_TIMERHASERROR},
-    {"timerhasconflict", LISTITEM_TIMERHASCONFLICT},
-    {"addonname", LISTITEM_ADDON_NAME},
-    {"addonversion", LISTITEM_ADDON_VERSION},
-    {"addoncreator", LISTITEM_ADDON_CREATOR},
-    {"addonsummary", LISTITEM_ADDON_SUMMARY},
-    {"addondescription", LISTITEM_ADDON_DESCRIPTION},
-    {"addondisclaimer", LISTITEM_ADDON_DISCLAIMER},
-    {"addonnews", LISTITEM_ADDON_NEWS},
-    {"addonbroken", LISTITEM_ADDON_BROKEN},
-    {"addonlifecycletype", LISTITEM_ADDON_LIFECYCLE_TYPE},
-    {"addonlifecycledesc", LISTITEM_ADDON_LIFECYCLE_DESC},
-    {"addontype", LISTITEM_ADDON_TYPE},
-    {"addoninstalldate", LISTITEM_ADDON_INSTALL_DATE},
-    {"addonlastupdated", LISTITEM_ADDON_LAST_UPDATED},
-    {"addonlastused", LISTITEM_ADDON_LAST_USED},
-    {"addonorigin", LISTITEM_ADDON_ORIGIN},
-    {"addonsize", LISTITEM_ADDON_SIZE},
-    {"expirationdate", LISTITEM_EXPIRATION_DATE},
-    {"expirationtime", LISTITEM_EXPIRATION_TIME},
-    {"art", LISTITEM_ART},
-    {"property", LISTITEM_PROPERTY},
-    {"parentalrating", LISTITEM_PARENTAL_RATING},
-    {"parentalratingcode", LISTITEM_PARENTAL_RATING_CODE},
-    {"currentitem", LISTITEM_CURRENTITEM},
-    {"isnew", LISTITEM_IS_NEW},
-    {"isboxset", LISTITEM_IS_BOXSET},
-    {"totaldiscs", LISTITEM_TOTALDISCS},
-    {"releasedate", LISTITEM_RELEASEDATE},
-    {"originaldate", LISTITEM_ORIGINALDATE},
-    {"bpm", LISTITEM_BPM},
-    {"uniqueid", LISTITEM_UNIQUEID},
-    {"bitrate", LISTITEM_BITRATE},
-    {"samplerate", LISTITEM_SAMPLERATE},
-    {"musicchannels", LISTITEM_MUSICCHANNELS},
-    {"ispremiere", LISTITEM_IS_PREMIERE},
-    {"isfinale", LISTITEM_IS_FINALE},
-    {"islive", LISTITEM_IS_LIVE},
-    {"tvshowdbid", LISTITEM_TVSHOWDBID},
-    {"albumstatus", LISTITEM_ALBUMSTATUS},
-    {"isautoupdateable", LISTITEM_ISAUTOUPDATEABLE},
-    {"hdrtype", LISTITEM_VIDEO_HDR_TYPE},
+const infomap listitem_labels[]= {{ "thumb",            LISTITEM_THUMB },
+                                  { "icon",             LISTITEM_ICON },
+                                  { "actualicon",       LISTITEM_ACTUAL_ICON },
+                                  { "overlay",          LISTITEM_OVERLAY },
+                                  { "label",            LISTITEM_LABEL },
```

Comment:
This blows git history for a bunch of lines, better to add `//clang-format off` before and `//clang-format on` after?